VTEX - TEMPLATE.json

```diff
@@ -0,0 +1,236 @@
+                    "description": "Description of environment used in API.",
+                    "default": "{environment}"
+                }
+            }
+        }
+    ],
+    "paths": {
+        "/pets": {
+            "get": {
+                "summary": "List all pets",
+                "operationId": "listPets",
+                "description": "Describe the purpose of this path and who should use it.",
+                "tags": [
+                    "pets"
+                ],
+                "parameters": [
+                    {
+                        "name": "Content-Type",
+                        "in": "header",
+                        "description": "Describes the type of the content being sent.",
+                        "required": true,
+                        "style": "simple",
+                        "schema": {
+                            "type": "string",
+                            "default": "application/json"
+                        }
+                    },
+                    {
+                        "name": "Accept",
+                        "in": "header",
+                        "description": "HTTP Client Negotiation _Accept_ Header. Indicates the types of responses the client can understand.",
+                        "required": true,
+                        "style": "simple",
+                        "schema": {
+                            "type": "string",
+                            "default": "application/json"
+                        }
+                    },
+                    {
+                        "name": "limit",
+                        "in": "query",
+                        "description": "How many items to return at one time (max 100)",
+                        "required": false,
+                        "schema": {
+                            "type": "integer",
+                            "example": "50"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An paged array of pets",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Pets"
+                                }
+                            }
+                        }
+                    }
+                },
+                "deprecated": false
+            },
+            "post": {
+                "summary": "Create a pet",
+                "operationId": "createPets",
+                "description": "Describe the purpose of this path and who should use it",
+                "tags": [
+                    "pets"
+                ],
+                "requestBody": {
+                    "description": "In the requestBody will be informed the data for insertion of new pets in the adoption list. This data must be sent in JSON format.",
```

Comment:
```suggestion
                    "description": "In the requestBody will be informed of the data for insertion of new pets in the adoption list. This data must be sent in JSON format.",
```

Reply:
Não sei se deixaria a frase `This data must be sent in JSON format.` porque nem sempre é o caso. Acho que talvez colocar **This data usually is an `application/json`.**